static/js/util/courseList.js

```diff
@@ -84,7 +87,10 @@ export function makeCourseStatusDisplay(course: Object, now: moment = moment()):
     } else {
       if (firstRun.course_id) {
         let courseInfoUrl = `${SETTINGS.edx_base_url}/courses/${firstRun.course_id}/about`;
-        return <Button bsStyle="success" href={courseInfoUrl} target="_blank">ENROLL</Button>;
+        return <Button bsStyle="success" href={courseInfoUrl} target="_blank">
+          ENROLL
+          <span className="sr-only"> in {firstRun.title}</span>
+        </Button>;
```

Comment:
When I actually inspect the markup it looks something like this:

![react-silliness](https://cloud.githubusercontent.com/assets/6207644/16122308/9850da08-33b4-11e6-9f0b-cfb9e0eb6b3b.png)

I imagine any screenreader will ignore comments, but do you think if we did this:

```js
<span className="sr-only">{` in ${firstRun.title}`}</span>
```

we'd have cleaner markup on the page?